.changesets/feat_feature_shaika_add_ms_support_to_rhai_engine.md

```diff
@@ -0,0 +1,4 @@
+### Support unix_ms_now in rhai engine ([Issue #5182](https://github.com/apollographql/router/issues/5182))
+
+
+By [@shaikatzz](https://github.com/shaikatzz) in https://github.com/apollographql/router/pull/5181
```

Comment:
```suggestion
### Add support for `unix_ms_now` in Rhai customizations ([Issue #5182](https://github.com/apollographql/router/issues/5182))

Rhai customizations can now use the `unix_ms_now()` function to obtain the current Unix timestamp in milliseconds since the Unix epoch. 

For example:

```rhai
fn supergraph_service(service) {
    let now = unix_ms_now();
}
```


By [@shaikatzz](https://github.com/shaikatzz) in https://github.com/apollographql/router/pull/5181
```